### ISSUE TITLE
Trouble following packaging libraries tutorial: --no-warn-script-location not a supported option

### ISSUE THREAD

**Issue by rtweeks** [pypa/packaging-problems#447]
Thank you for providing feedback on Python packaging!

To help us help you, please fill out as much of the following as you can. If a question is not relevant, feel free to skip it.

1. What is your operating system and version?

Ubuntu 18.04

2. What is your Python version?

Python 3.6.9

3. What version of pip do you have?

This is tricky.  I have `--user` installed pip 21.0.1, but system-wide it is pip 9.0.1 (which comes from the Ubuntu 18.04 `python3-pip` package).

4. If following an online tutorial or guide, please provide a link to the page or section giving you trouble:

https://packaging.python.org/tutorials/packaging-projects/

5. Could you describe your issue in as much detail as possible?

```
no such option: --no-warn-script-location
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/restorable/richardt/boto3-mocking/pyenv/lib/python3.6/site-packages/build/__main__.py", line 214, in <module>
    main(sys.argv[1:], 'python -m build')
  File "/home/restorable/richardt/boto3-mocking/pyenv/lib/python3.6/site-packages/build/__main__.py", line 206, in main
    build_package(args.srcdir, outdir, distributions, config_settings, not args.no_isolation, args.skip_dependencies)
  File "/home/restorable/richardt/boto3-mocking/pyenv/lib/python3.6/site-packages/build/__main__.py", line 94, in build_package
    _build_in_isolated_env(builder, outdir, distributions, config_settings)
  File "/home/restorable/richardt/boto3-mocking/pyenv/lib/python3.6/site-packages/build/__main__.py", line 56, in _build_in_isolated_env
    env.install(builder.build_dependencies)
  File "/home/restorable/richardt/boto3-mocking/pyenv/lib/python3.6/site-packages/build/env.py", line 143, in install
    subprocess.check_call(cmd)
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/tmp/build-env-vpzqtkm3/bin/python', '-Im', 'pip', 'install', '--no-warn-script-location', '-r', '/tmp/build-reqs-21oztz8t.txt']' returned non-zero exit status 2.
```

**Comment by henryiii** [comment]
Known bug, a new release of build should be out soon! Fixed in https://github.com/pypa/build/pull/229

**Comment by henryiii** [comment]
(venv contains a copy of pip 9, which is what it's injecting into the new build environment that build is using.)

**Comment by henryiii** [comment]
Fixed in build 0.3.0! Please reopen if there are still issues.
